### PR TITLE
i18n(fr): Add `guides/view-transitions` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `guides/view-transitions` in French language
#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/view-transitions.mdx [hunk 1]
```diff
@@ -0,0 +1,721 @@
...
+  () => window.scrollTo({ left: 0, top: 0, behavior: 'instant' }))
+```
+
+### `astro:page-load`
+
+Un événement qui se déclenche à la fin de la navigation de la page, après que la nouvelle page soit visible par l'utilisateur et que les styles et les scripts bloquants soient chargés. Vous pouvez écouter cet événement sur le `document`.
+
+Le composant `<ViewTransitions />` déclenche cet événement à la fois lors de la navigation initiale sur une page pré-rendue et lors de toute navigation ultérieure, que ce soit en avant ou en arrière.
+
+You can use this event to run code on every page navigation, or only once ever:
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/view-transitions.mdx:591]:
```suggestion
Vous pouvez utiliser cet événement pour exécuter du code à chaque navigation de page, ou seulement une fois par jour :
```

**Review by McFlyPartages — COMMENTED** [review]
A few suggestions and omissions. Thanks for work

**Comment by thomasbnt** [comment]
> A few suggestions and omissions. Thanks for work

Thanks @McFlyPartages for reviews! 🚀

**Review by McFlyPartages — APPROVED** [review]
LGTM

**Pull request merged**